prime-router/examples/generate-jwt-python/jwt-errors.md

```diff
@@ -0,0 +1,27 @@
+# Errors when generating a signed JWT
+As part of [the authentication process](https://reportstream.cdc.gov/resources/api/getting-started#set-up-authentication), an organization needs to generate a signed JSON Web Token (JWT). The signed JWT is used to retrieve an access token. 
```

Comment:
Great doc, thanks!

Suggestion: use "access (bearer) token" here to match the referenced setup doc.

Reply:
Thanks! Updated